src/components/Navbar/Navbar.css

```diff
@@ -2,6 +2,10 @@
   width: 18rem;
 }
 
+.margin {
+  margin-bottom: 5em;
+}
+
```

Comment:
can you change the value for 'rem'? and you can be more specific with the classnames, like: mb-0-m for mediaquery, and mb-5.
(the '-' is not to be stepped on with tachyons classes.)